content/docs/command-reference/get-url.md

```diff
@@ -73,19 +82,6 @@ $ wget https://example.com/path/to/data.csv
 
 <details>
 
-### Click and expand for a local example
-
-```dvc
-$ dvc get-url /local/path/to/data
-```
-
-The above command will copy the `/local/path/to/data` file or directory into
-`./dir`.
-
-</details>
-
-<details>
-
 ### Click for Amazon S3 example
```

Comment:
Reordered this local example same as in remote add/modify refs (put in the bottom).